src/evaluators/ObjectExpression.js

```diff
@@ -24,21 +25,31 @@ import {
   HasOwnProperty,
   PropertyDefinitionEvaluation,
   ToPropertyKey,
-  ToStringPartial
+  ToString
 } from "../methods/index.js";
 import invariant from "../invariant.js";
 import type { BabelNodeObjectExpression, BabelNodeObjectProperty, BabelNodeObjectMethod, BabelNodeClassMethod } from "babel-types";
 
 // Returns the result of evaluating PropertyName.
-export function EvalPropertyName(prop: BabelNodeObjectProperty | BabelNodeObjectMethod | BabelNodeClassMethod, env: LexicalEnvironment, realm: Realm, strictCode: boolean): PropertyKeyValue {
+export function EvalPropertyNamePartial(prop: BabelNodeObjectProperty | BabelNodeObjectMethod | BabelNodeClassMethod, env: LexicalEnvironment, realm: Realm, strictCode: boolean): PropertyKeyValue {
```

Comment:
If this is a common pattern, maybe there's a way to factor it out so that you don't have to keep retyping it, but not sure if that's possible while retaining Flow checking.